ext/sidecar.c

```diff
@@ -77,20 +76,18 @@ ddog_SidecarTransport *dd_sidecar_connection_factory(void) {
                                     DDOG_CHARSLICE_C("php"),
                                     DDOG_CHARSLICE_C(PHP_DDTRACE_VERSION),
                                     get_global_DD_TRACE_AGENT_FLUSH_INTERVAL(),
+                                    (int)(get_global_DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS() * 1000),
                                     // for historical reasons in seconds
                                     get_global_DD_TELEMETRY_HEARTBEAT_INTERVAL() * 1000,
                                     get_global_DD_TRACE_BUFFER_SIZE(),
                                     get_global_DD_TRACE_AGENT_STACK_BACKLOG() * get_global_DD_TRACE_AGENT_MAX_PAYLOAD_SIZE(),
                                     get_global_DD_TRACE_DEBUG() ? DDOG_CHARSLICE_C("debug") : dd_zend_string_to_CharSlice(get_global_DD_TRACE_LOG_LEVEL()),
                                     (ddog_CharSlice){ .ptr = logpath, .len = strlen(logpath) },
-
-                                    // Not used yet
-                                    NULL,
-                                    remote_config_products,
-                                    0,
-                                    remote_config_capabilities,
-                                    0
-                                    );
+                                    ddtrace_set_all_thread_vm_interrupt,
+                                    DDTRACE_REMOTE_CONFIG_PRODUCTS,
+                                    sizeof(DDTRACE_REMOTE_CONFIG_PRODUCTS) / sizeof(DDTRACE_REMOTE_CONFIG_PRODUCTS[0]) - !get_DD_DYNAMIC_INSTRUMENTATION_ENABLED(),
```

Comment:
It's probably worth adding a comment where `DDTRACE_REMOTE_CONFIG_PRODUCTS` is declared, indicating that `RemoteConfigProduct::LiveDebugger` must remain the last element.

Reply:
Agree.